app/components/documents/document-card.js

```diff
@@ -57,22 +57,21 @@ export default class DocumentsDocumentCardComponent extends Component {
 
   @task
   *loadPieceRelatedData() {
+    const loadPiece = (id) => this.store.queryOne('piece', {
+      'filter[:id:]': id,
+      include: 'document-container,document-container.type,access-level,publication-flow,publication-flow.identification',
+    });
```

Comment:
Deze include aanpassen met publicatie specifieke eigenschappen maakt deze component minder generiek. Ik denk niet dat de approach is dat we hier willen ?

Reply:
De replace van de 2 query's naar 1 task query is wel een goede change

Reply:
@ValenberghsSven Ik snap uw bezorgdheid. Uit het design is het mij niet helemaal duidelijk, maar ik denk dat voor mensen van OVRB (en admin) die link naar de publication-flow, als er een is, op iedere document card getoond moet worden. Dus mits het toevoegen van een conditie dat dit enkel getoond wordt voor OVRB en admin users, lijkt me het wel ok om dat hier toe te voegen. Dan kunnen we nog opteren om de `publication-flow` niet mee in de `include` te steken zodanig dat de query niet zwaarder wordt voor niet-OVRB users. En bij OVRB users zullen er dan 2 extra network requests getriggerd worden vanuit de template voor iedere document card.

Had je iets specifiek in gedachte hoe we het anders kunnen oplossen? Het enige alternatief dat ik nu kan bedenken is een nieuwe component `document-publication-card` maken die overerft van de `document-card` component, maar dan gaat het gros van de template gewoon gedupliceerd worden met een grote kans dat die out-of-sync gaan geraken op termijn.

(CC @MikiDi Dit interesseert je waarschijnlijk wel)

Reply:
Probleem is dat ik zelf niet goed kan zeggen hoe we dit beter doen.
Daar is de structuur van die documentCardList > documentList > documentCard iets te complex, anders zouden we misschien extra toolbar opties kunnen yielden, maar dan zit de data loading nog niet goed natuurliijk...

Als de link naar publicatie een vast item wordt in de document card, dan hoort het hier wel thuis.
Met de afscherming van de feature flag / ovrb beperking zijn de network calls al minder zwaar voor niet ovrb users

Eventueel nog een argument kunnen meegeven van de top level om dit uit te schakelen (in sommige documenten views wil je die knop misschien sowieso niet, ongeacht of het ovrb is of niet)

Kan in een vervolg story, deze staat nu al een tijdje open